xml/transactional_updates.xml

```diff
@@ -0,0 +1,517 @@
+    <term><literal>dup</literal></term>
+    <listitem>
+     <para>
+      If there are new updates available, a new snapshot is created and
+      <command>zypper dup –no-allow-vendor-change</command> updates the
+      snapshot. The snapshot is activated afterwards and becomes the new root
+      file system after reboot.
+     </para>
+<screen>&prompt.root;<command>transactional-update dup</command></screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><literal>patch</literal></term>
+    <listitem>
+     <para>
+      If there are new updates available, a new snapshot is created and
+      <command>zypper patch</command> updates the snapshot.
+     </para>
+<screen>&prompt.root;<command>transactional-update patch</command></screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><literal>rollback</literal></term>
+    <listitem>
+     <para>
+      This sets the default subvolume. On systems with a read-write file system
+      <command>snapper rollback</command> is called. On a read-only file system
+      and without any argument, the current system is set to a new default root
+      file system. If you specify a number, that snapshot is used as the
+      default root file system. On a read-only file system, it does not create
+      any additional snapshots.
+     </para>
+<screen>&prompt.root;<command>transactional-update rollback <replaceable>snapshot_number</replaceable></command></screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><literal>grub.cfg</literal></term>
+    <listitem>
+     <para>
+      This creates a new GRUB2 configuration. Sometimes it is necessary to
+      adjust the boot configuration, for example adding additional kernel
+      parameters. Edit <replaceable>/etc/default/grub</replaceable>, run
+      <command>transactional-update grub.cfg</command>, and then reboot to
+      activate the change. You must immediately reboot, or the new GRUB2
+      configuration will be overwritten with the default by the next
+      transactional-update.
+     </para>
+<screen>&prompt.root;<command>transactional-update grub.cfg</command></screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><literal>reboot</literal></term>
+    <listitem>
+     <para>
+      This parameter triggers a reboot after the action is completed.
+     </para>
+<screen>&prompt.root;<command>transactional-update <replaceable>dup</replaceable> reboot</command></screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><literal>--help</literal></term>
+    <listitem>
+     <para>
+      This prints a help screen with options and subcommands.
+     </para>
+<screen>&prompt.root;<command>transactional-update --help</command></screen>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+ </sect1>
+ <sect1 xml:id="sec.tu-disable">
+  <title>Disabling Transactional Updates</title>
+
+  <para>
+   You can disable transactional updates with this command:
```

Comment:
I would rename the section to: Automatic Updates

and explain, that a systemd-timer will run once a day, applies all updates, and informs the rebootmgrd that the machine should be rebooted.
To diable automatic updates, run "systemctl ...". The time, when the update runs, can be adjusted as needed, see systemd-timer manual page. The maintenance window, in which rebootmgrd will reboot the machine, is also configureable, see rebootmgrd manual page.